create-env.bat

```diff
@@ -59,10 +59,13 @@ REM
     REM Reset errorlevel to 0
     VERIFY > nul
 
-    CALL conda --info REM >nul 2>nul
+    CALL conda info 1>nul 2>nul
     IF NOT ERRORLEVEL 1 (
         where conda > .conda_path
         SET /p CONDA_PATH= < .conda_path
+		SET CONDA_PATH=!CONDA_PATH:Library\bin=Scripts!
+		SET CONDA_PATH=!CONDA_PATH:conda.bat=!
+		SET CONDA_PATH=!CONDA_PATH:conda.exe=!
```

Comment:
This is not beautiful but a solution that should apply to both cases mentioned before